### PR TITLE
Encrypt thumbnails uploaded by a client.

### PR DESCRIPTION
**Pull request by aem09**
If a client supplies a thumbnail when sending some media into a room, pantalaimon does not currently add the information to decrypt the thumbnail to the outgoing message - meaning that the thumbnail is not displayed.

This fixes that problem by adding the thumbnail_file key in the same way that the file key is added to the outgoing message.

I wasn't sure of the most sensible way to do this (I'm not very familiar with pantalaimon's codebase), but the method I've used seems to work for me! However it's quite possible I've missed or misunderstood something.

### CODE REVIEW

**Comment by aem09** [comment]
For clarity, here is a before/after of the difference when sending a video with a thumbnail through pantalaimon to element desktop:

image

File: pantalaimon/store.py [hunk 1]
```diff
@@ -60,6 +60,19 @@ def to_content(self, content: Dict, mime_type: str) -> Dict[Any, Any]:
 
         return content
 
+    def to_thumbnail(self, content: Dict, mime_type: str) -> Dict[Any, Any]:
```
Comment (poljar) [inline pantalaimon/store.py:63]: It's a bit weird that this takes the content, modifies it and returns it. Maybe just remove the return.

**Review by poljar — CHANGES_REQUESTED** [review]
Looks mostly good, a small nit about the way we add the thumbnail info.

File: pantalaimon/store.py (hunk 1 above)
Comment (aem09) [reply pantalaimon/store.py:63]: Makes sense. I did that as a direct copy of the MediaInfo.to_content() method, I'll remove the return from both of them(?)

Comment (poljar) [reply pantalaimon/store.py:62]: Sure.

Comment (aem09) [reply pantalaimon/store.py:62]: Thanks, that's done in the latest commit!

**Comment by poljar** [comment]
Thanks, merged.

**Pull request merged**